src/components/CartPage/CartListItem/CartListItemView.tsx

```diff
@@ -1,33 +1,61 @@
 import './cartListItem.scss';
 
-function CartListItemView() {
+interface CartListItemViewProps {
+  title: string | undefined;
+  description: string | undefined;
+  rating: number | undefined;
+  discount: number | undefined;
+  stock: number | undefined;
+  price: number | undefined;
+  thumbnail: string | undefined;
+  position: number | undefined;
+  id: number | undefined;
```

Comment:
I think it will be better to use optional value instead union type with undefined